src/main.ts

```diff
@@ -40,7 +31,7 @@ async function mapiCLI(): Promise<string> {
 
```

Comment:
I think the cache key above and also later in this function needs to change from `'mapi'` to something like `'mapi-platform`', so we don't accidentally use a cached version from legacy mapi, which is built with different features.

Reply:
Changed cache key to `security.mayhem.app.cli.mapi` in c272f9674c7b14d3766725835d2f08e04924fc28 #Java